metadata.yaml

```diff
@@ -1,5 +1,5 @@
 homepage: "https://www.redditinc.com/advertising"
-documentation: "https://advertising.reddithelp.com/en/categories/measurement/install-reddit-pixel#googleTM"
+documentation: "https://reddit.my.site.com/helpcenter/s/article/Install-the-Reddit-Pixel-on-your-website"
```

Comment:
Bummer that there's no `id` on the `<h2>` for the "Install the pixel with Google Tag Manager" section.  Maybe ask Katie to add one so it can be used in the fragment here?

Reply:
Agreed! I'll ask